src/reports/report-acl.schema.json

```
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "The type of entity being granted access to this report (either \"user\" or \"role\")."
```

We should be able to dictate the enum for these options.

```suggestion
      "description": "The type of entity being granted access to this report (either \"user\" or \"role\").",
      "enum": ["user", "role"]
```